includes/abstracts/abstract-ur-field-settings.php

```diff
@@ -54,7 +54,7 @@ public function render_html( $fields ) {
 
 			$tooltip_html       = ! empty( $field['tip'] ) ? ur_help_tip( $field['tip'], false, 'ur-portal-tooltip' ) : '';
 			$this->fields_html .= '<div class="ur-advance-setting ur-advance-' . esc_attr( $field_key ) . '">';
-			$this->fields_html .= '<label for="' . esc_attr( $field['class'] ) . '">' . ( isset( $field['label'] ) ? esc_attr( $field['label'] ) : '' ) . $tooltip_html . '</label>';
+			$this->fields_html .= '<label for="ur-type-' . $field['type'] . '">' . ( isset( $field['label'] ) ? esc_attr( $field['label'] ) : '' ) . $tooltip_html . '</label>';
```

Comment:
@iamprazol Can you please let me know why did you change this code?

Reply:
This may have been gone there mistakely and as it had not serious side effects i was unaware of this change